libbeat/common/transport/tlscommon/config.go

```diff
@@ -0,0 +1,86 @@
+package tlscommon
```

Comment:
I assume you named it `tlscommon` instead of `tls` because of the name conflict?

Reply:
Exactly, I am open to better name.